device_config.sample.json

```diff
@@ -17,6 +17,16 @@
     }
   },
   "billDispenser": {
-    "device": null
+    "device": null,
+    "mockedBalance": {
```

Comment:
here you can force the balance

Reply:
Also, we want a way to force other errors. 

> Most common are the pickup errors such as 81 (0x51).